packages/agreement-process/test/agreementConsumerDocuments.test.ts

```diff
@@ -49,40 +50,51 @@ import {
 
 describe("agreement consumer document", () => {
   describe("get", () => {
-    let agreement1: Agreement;
```

Comment:
It was used only in half of the tests.
I've manually added it where it was needed